### DISCUSSION TITLE
Value Error: Units 'M', 'Y' and 'y'

### DISCUSSION THREAD

**Discussion by humbertodlltrr** [winedarksea/AutoTS#209] [Q&A]
I have monthly data that I am generating forecasts for, I have only found issues with `model.plot_validations()` function so far. It seems this end in an eventual `Timedelta(r, unit)` which is no longer supported for anything higher than days D. Is there a workaround at the moment that I haven't yet percieved?

**Comment by winedarksea** [comment]
I just ran a test on monthly data and plot_validations worked fine with pandas==2.1.3. Could you provide more details about your error?

**Reply by humbertodlltrr** [reply]
Im using the following:
python 3.10.0
pandas==2.1.3
autots==0.6.3

The call in particular is in evaluator\auto_ts.py, line 3142 where it evaluates my models frequency, M, into "1M"

I'm thinking its up to a dependency that causes in this virtual environment to take `pd.to_timedelta(used_freq)`, where used_freq is 1M, to the specified error. On my personal machine it seems to work fine but the same code is evaluated to `Timedelta('0 days 00:01:00')` which would still be an unwanted output.

**Reply by winedarksea** [reply]
I've got it now, thanks for the details: ValueError: Units 'M', 'Y' and 'y' do not represent unambiguous timedelta values and are not supported.

I'll try to put together a fix for the next release

**Reply by winedarksea** [reply]
I have pushed a fix to dev.
 For now, if you need an immediate fix. Try setting `start_date="2023-01-01"`, or some other date, in plot_validations, which will bypass the bug
